boilerplate/aom/logo.include

```diff
@@ -1,3 +1,3 @@
 <a href="https://www.aomedia.org/" class="logo">
-    <img alt="AOM" src="https://aomedia.org/assets/images/aomedia-icon-only.png" width="174" height="174">
+    <img alt="AOM" src="https://aomedia.org/assets/images/aomedia-logo-rgb.svg" width="200">
```

Comment:
Might as well provide the height as well, since you have that information. Looking at the file itself, the native aspect ratio is 3.375 (a hair over 10/3), so a height of 59 or 60 is appropriate.

Reply:
done